modules/images/dominant-color/load.php

```diff
@@ -303,15 +274,25 @@ function dominant_color_has_transparency( $attachment_id ) {
 	if ( is_wp_error( $editor ) ) {
 		return $editor;
 	}
+
 	if ( ! method_exists( $editor, 'has_transparency' ) ) {
 		return new WP_Error( 'unable_to_find_method', __( 'Unable to find has_transparency method', 'performance-lab' ) );
 	}
 	$has_transparency = $editor->has_transparency();
-	if ( is_wp_error( $has_transparency ) ) {
-		return $has_transparency;
+	if ( ! is_wp_error( $has_transparency ) ) {
+		$dominant_color_data['has_transparency'] = $has_transparency;
+	}
```

Comment:
Why are we ignoring this error here? This also makes the code much harder to test.